packages/manager/modules/dbaas-logs/src/logs/detail/streams/subscriptions/streams-subscriptions.controller.js

```diff
@@ -33,6 +33,24 @@ export default class LogsStreamsSubscriptionsCtrl {
     this.stream.load();
   }
 
+  getResourceName(subscription) {
+    const key = `streams_subscriptions_resource_products_${subscription.resource.type}`;
+    const translated = this.$translate.instant(key);
+    if (translated === key) {
+      const resourceName = subscription.resource.type
+        .replace('-', ' ')
+        .toLowerCase()
+        .split(' ');
+      for (let i = 0; i < resourceName.length; i += 1) {
+        resourceName[i] =
+          resourceName[i].charAt(0).toUpperCase() +
+          resourceName[i].substring(1);
+      }
```

Comment:
**nitpick:** Use `lodash.capitalize` instead
https://lodash.com/docs#capitalize

Reply:
Yes it do the job, thank you for the tip. Code has been updated.